python_src/src/lamp_py/performance_manager/flat_file.py

```diff
@@ -127,7 +127,7 @@ def write_flat_files(db_manager: DatabaseManager) -> None:
 
     for service_date in service_dates:
         sub_process_logger = ProcessLogger(
-            "flat_file_write", service_date=service_date
+            "flat_file_write", service_date=service_date.strftime("%Y-%m-%d")
```

Comment:
this was writing logs with a space between the date and time, triggering logging errors.